### PR TITLE
feat: update aws-network-acl-allows-all-ports to cover aws_default_network_acl resources

### PR DESCRIPTION
**Pull request by gabriellesc**
## Description

This PR updates the `terraform.aws.security.aws-network-acl-allows-all-ports` rule to cover the `aws_default_network_acl` resource (previously, it only covered `aws_network_acl` and `aws_network_acl_rule`).

### CODE REVIEW

File: terraform/aws/security/aws-network-acl-allows-all-ports.yaml [hunk 1]
```diff
@@ -29,7 +29,10 @@ rules:
               action = "allow"
               ...
             }
-  - metavariable-regex:
+      - metavariable-regex:
+          metavariable: $NETWORK_ACL
+          regex: ^(aws_network_acl|aws_default_network_acl)$
+    - metavariable-regex:
```
Comment (kurt-r2c) [inline terraform/aws/security/aws-network-acl-allows-all-ports.yaml]: this has broken indentation which is causing tests to fail, should be 

```yaml
metavariable-regex:
  metavariable: $PROTOCOL
  ...
```

Comment (gabriellesc) [reply terraform/aws/security/aws-network-acl-allows-all-ports.yaml]: Fixed, thank you for pointing that out!

**Review by enncoded — APPROVED** [review]

File: terraform/aws/security/aws-network-acl-allows-all-ports.yaml [hunk 2]
```diff
@@ -33,8 +33,8 @@ rules:
           metavariable: $NETWORK_ACL
           regex: ^(aws_network_acl|aws_default_network_acl)$
     - metavariable-regex:
```
Comment (enncoded) [inline terraform/aws/security/aws-network-acl-allows-all-ports.yaml]: Ah one more thing, you need to un-indent the last `metavariable-regex` as a whole (L35-37) by 2 spaces

**Review by enncoded — CHANGES_REQUESTED** [review]

File: terraform/aws/security/aws-network-acl-allows-all-ports.yaml [hunk 3]
```diff
@@ -29,9 +29,12 @@ rules:
...
             }
-  - metavariable-regex:
-      metavariable: $PROTOCOL
-      regex: \A("all"|"-1"|-1)\Z
+      - metavariable-regex:
+          metavariable: $NETWORK_ACL
+          regex: ^(aws_network_acl|aws_default_network_acl)$
+    - metavariable-regex:
+        metavariable: $PROTOCOL
+        regex: \A("all"|"-1"|-1)\Z
```
Comment (enncoded) [inline terraform/aws/security/aws-network-acl-allows-all-ports.yaml]: Ah one more thing, you need to un-indent the last `metavariable-regex` as a whole (L35-37) by 2 spaces

**Review by enncoded — CHANGES_REQUESTED** [review]

File: terraform/aws/security/aws-network-acl-allows-all-ports.yaml (hunk 3 above)
Comment (gabriellesc) [reply terraform/aws/security/aws-network-acl-allows-all-ports.yaml]: This is what I get for making changes in the GitHub editor! 😅 
Fixed, sorry about that.

Comment (enncoded) [reply terraform/aws/security/aws-network-acl-allows-all-ports.yaml]: All good!!

**Review by enncoded — APPROVED** [review]

**Comment by enncoded** [comment]
LGTM

**Pull request merged**